cmd/podman/containers/top.go

```diff
@@ -91,6 +91,9 @@ func top(cmd *cobra.Command, args []string) error {
 	if len(args) < 1 && !topOptions.Latest {
 		return errors.Errorf("you must provide the name or id of a running container")
 	}
+	if len(args) > 0 && topOptions.Latest {
+		return errors.Errorf("--latest and containers cannot be used together")
+	}
```

Comment:
`podman top` shouldn't check this. This is valid:
```
$ bin/podman top -l pid user
PID   USER
1     root
```

Reply:
Good catch, thank you. Fixed.